ol-keycloak/oltheme/src/main/resources/theme/ol/login/messages/messages_en.properties

```diff
@@ -1,7 +1,7 @@
 noAccount=Don''t have an account?
 doRegister=Sign up
 doRegisterSubmit=Next
-loginAccountTitle=Sign In
+loginAccountTitle=Log in
```

Comment:
Will you also need to configure the following messages?:

1. expiredCodeMessage
2. staleCodeMessage
3. password-help-text
4. auth-username-form-help-text
5. auth-username-password-form-help-text

Reply:
I pulled in all the default values, commented them out, and then replaced the ones we override. Hopefully this way it's harder to miss some of these when they don't show up in templates we override or are used by backend code.

Reply:
I would probably keep the changes to a minimum in order to support future upgrades of Keycloak.  I don't think keeping our local copy up to date with the OOTB copy will be easy.